library/init/meta/interactive.lean

```diff
@@ -133,813 +132,3 @@ meta def param_desc : expr → tactic format
   then return $ to_fmt "{ tactic }"
   else paren <$> pp e
 end interactive
-
```

Comment:
We could move the `*_tactic(s).lean` files into the directory. However, your suggestion is probably better for the time being since it avoids conflicts with other open PRs. I will do the change.